modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/utils/constants/subscriptionCardsCount.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+type ArrayOfString = {
+	[key: string]: string[];
+};
+
+export const SUBSCRIPTION_TYPES: ArrayOfString = {
+	Blank: ['Partnership', 'Others'],
+	Purchased: [
+		'Enterprise Search',
+		'LXC - SM',
+		'Analytics Cloud',
+		'Commerce for Cloud',
+		'Commerce',
+		'Liferay Experience Cloud',
+	],
+	PurchasedAndProvisioned: ['Portal', 'DXP'],
+};
+
+export const PRODUCT_DISPLAY_EXCEPTION: ArrayOfString = {
+	productPurchased: ['Extended Premium Support'],
+	productsBlank: [
+		'CSP - Custom',
+		'CSP - Up ',
+		'Pro',
+		'Business',
+		'Enterprise',
+	],
+	productsNotBlank: ['Contact', 'Mobile Device'],
```

Comment:
I think it could be better, but I don't know the better way. Maybe it?
```suggestion
	nonBlankProducts: ['Contact', 'Mobile Device'],
```